### PR TITLE
Docs for DRF Plugin

### PR DESCRIPTION
**Pull request by Rajadeepan**
**What this PR does / why we need it**:
   Added a doc for DRF implemention in kube batch

Preview :  https://github.com/Rajadeepan/kube-batch/blob/doc/doc/design/plugin/drf.md

https://github.com/kubernetes-sigs/kube-batch/pull/703

### CODE REVIEW

File: doc/design/plugin/drf.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Kube-Batch Implementation
+DRF calculate shares for each job. The share is the highest value of  ratio of the (allocated resource/Total Resource) of the three resource types CPU, Memory and GPU.
+This share value is used for job ordering and task premption.
+
+#### Job Ordering:
+  The job having the lowest share will have higher priority.
+  In the example below all the tasks task1, task2 of job1 and task3 and task4 of job2 is already allocated to the cluster.  	  	
+  ![drfjobordering](drfjobordering.png)
+
+
```
Comment (k82cn) [inline doc/design/plugin/drf.md:21]: please also include how DRF work with minAvailable.

File: doc/design/plugin/drf.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+This share value is used for job ordering and task premption.
+
+#### 1. Job Ordering:
+  The job having the lowest share will have higher priority.
+  In the example below all the tasks task1, task2 of job1 and task3 and task4 of job2 is already allocated to the cluster.  	  	
+  ![drfjobordering](drfjobordering.png)
+
+
+ ##### 1.1 Gang Scheduling with DRF in job ordering ( Gang -> DRF) <br />
+   Gang scheduling sorts the job based on whether the job has atleast <b>minAvailable</b> task already (allocated + successfully completed + pipelined) or not.
```
Comment (k82cn) [inline doc/design/plugin/drf.md]: please use markdown instead of html code.

Comment (Rajadeepan) [reply doc/design/plugin/drf.md]: updated with markdown

**Pull request merged**